src/fs/root/tree.ts

```diff
@@ -308,8 +310,15 @@ function loadPrivateTrees(
         const bareNameFilter = await findBareNameFilter(map, path)
         if (!bareNameFilter) throw new Error(`Was trying to load the PrivateTree for the path \`${path}\`, but couldn't find the bare name filter for it.`)
 
-        privateTree = await PrivateTree.fromBareNameFilter(mmpt, bareNameFilter, key)
-
+        const maybeInfo = await protocol.priv.getLatestByBareNameFilter(mmpt, bareNameFilter, key)
+        if (maybeInfo === null) throw new Error(`Could not find content in filesystem for path ${path}`)
+        if(check.isPrivateTreeInfo(maybeInfo)) {
```

Comment:
Is there a reason you prefer this over just using `pathing.isFile(path)`?

Reply:
:monocle_face: 

nope I don't think so. I'm just not super used to the new pathing tools so didn't think of it :ok_hand: